smarts/core/mission_planner.py

```diff
@@ -62,6 +62,7 @@ def __init__(
         self._prev_kyber_x_position = None
         self._prev_kyber_y_position = None
         self._first_uturn = True
+        self._uturn_target_velocity_window = [0] * 10
```

Comment:
this seems no longer used

Reply:
Yes, I saw this earlier and forgot to remove it.